# Accelerometer [m/s^2]
geometry_msgs/Vector3 accel

# Gyroscope [rad/s]
geometry_msgs/Vector3 gyro

# Internal Temperature [Â°C]
float64 temp

int32 count
